ci/docker/runtime_functions.sh

```diff
@@ -762,6 +765,7 @@ build_ubuntu_gpu_cuda101_cudnn7_make() {
     make \
         USE_BLAS=openblas                         \
         USE_MKLDNN=0                              \
+        USE_TVM_OP=1                              \
```

Comment:
@leezu are the make builds still in use?

Reply:
They were kept as various language bindings hadn't been updated for the cmake build. As those were removed recently, we can delete the Makefile without impacting the CI